tests/ios/xcodeproj/BUILD.bazel

```diff
@@ -186,3 +186,15 @@ xcodeproj(
         "//tests/ios/app:FW",
     ],
 )
+
+xcodeproj(
+    name = "Test-Custom-Output-Path",
+    additional_lldb_settings = [
+        "log enable -f /tmp/lldb.log lldb types",
+    ],
```

Comment:
So you're using an existing proj so you can diff the paths at the end right?

Reply:
Yep. Since I cant test the project properly because current test infrastructure depends on having the xcode projects at `tests/ios/xcodeproj` (see issue description), I am using an alternative way of testing this. I am generating the same `Test-Custom-Output-Path` project under the `custom_output_path`, and then comparing the diff, which you can find in `tests/ios/xcodeproj/custom_output_path/test_expected_diff.txt `